R/mergeData.R

```diff
@@ -1,7 +1,11 @@
 mergeData <- function(newID, datList, oldIDs=NULL, addMbd = FALSE, verbose=TRUE) {
+  checkmate::assert_character(newID, len = 1)
+  checkmate::assert_list(datList, types = "data.frame")
```

Comment:
please use the same logic as in automateDataPreparation. It should be possible (even though not really useful but sometimes unavoidable in standard automation processes) to run the function with datList =singleDat